dist/stat_data.py

```diff
@@ -518,6 +518,8 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_checkpoint_fsync_post', 'transaction fsync calls for checkpoint after allocating the transaction ID'),
     TxnStat('txn_checkpoint_fsync_post_duration', 'transaction fsync duration for checkpoint after allocating the transaction ID (usecs)', 'no_clear,no_scale'),
     TxnStat('txn_checkpoint_handle_applied', 'transaction checkpoint most recent handles applied'),
+    TxnStat('txn_checkpoint_handle_ckptlist_meta', 'transaction checkpoint most recent handles read checkpoint list from metadata'),
+    TxnStat('txn_checkpoint_handle_ckptlist_saved', 'transaction checkpoint most recent handles used cached checkpoint list'),
```

Comment:
I'm trying hard to figure out why anybody not debugging this change would find these statistics useful? Do we need to leave them in the system?

Reply:
Resolving as stats will be removed.